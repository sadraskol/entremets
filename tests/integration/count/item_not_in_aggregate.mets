init do
    `insert into users (id, age) values (1, 10), (2, 20), (3, 30), (4, 40), (5, 50), (6, 60)`
end

process do
    `select count(*), id from users`
end
