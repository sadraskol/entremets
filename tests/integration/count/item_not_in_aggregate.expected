Error at tests/integration/count/item_not_in_aggregate.mets:6:26: Column id must appear in group by